test/gke/stop-registry.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+docker kill "$(cat ${script_dir}/registry_container)"
```

Comment:
`docker kill -f`?

Reply:
There is not `-f` flag for `docker kill` :(.

Reply:
You could `docker rm -f $(cat .../registry_container)`.